template/.github/workflows/preview.yml

```diff
@@ -0,0 +1,27 @@
+name: Netlify Preview
+on:
+  push:
```

Comment:
And should we add [workflow_dispatch](https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/) event to deploy manually from Github GUI?

Reply:
(note: comment assigned for the release job only :) )

Reply:
e146bdd ✅ 

Implemented for the `deploy.yml` action.

| Mannually run the workflow | Message is visible in Netlify |
|---|---|
|![image](https://user-images.githubusercontent.com/77609814/162151616-b1972181-66ed-4cc3-b6bf-acb810c76174.png)|![image](https://user-images.githubusercontent.com/77609814/162152130-b98a25ba-bc7f-4d5d-b566-ae2a1a14e205.png)|